### PR TITLE
Edit PingSource docs

### LINKED ISSUE #3507: [1.0] Fully document pingsources.sources.knative.dev
@n3wscott commented on [Mon Feb 22 2021](https://github.com/knative/eventing/issues/4916)

Related to https://github.com/knative/docs/issues/3268
The following work needs to be validated for pingsources.sources.knative.dev to be ready for 1.0:
- [x] Getting Started demo using the component.
- [x] Sample minimum of component
- [ ] Sample complete resource of component
- [ ] Fully documented features and options, including
    - [ ] Explanation of resource.
    - [ ] All spec fields
    - [ ] All status fields
    - [ ] Any magic labels or annotations that apply.
    - [x] With kubectl cli
    - [x] With kn cli

### PR DESCRIPTION
**Pull request by snneji**
Fixes #3507

### CODE REVIEW

File: docs/developer/eventing/sources/ping-source/README.md [hunk 1]
```diff
@@ -1,154 +1,226 @@
...
+            - `<sink-name>` is the name of your sink, for example, `http://event-display.pingsource-example.svc.cluster.local`.
+
+            For a list of available options, see the [Knative client documentation](https://github.com/knative/client/blob/main/docs/cmd/kn_source_ping_create.md).
+
+    === "YAML"
+
+        - To create a PingSource that sends data that can be represented as plain text,
+        such as text, JSON, or XML:
+
+            1. Create a YAML file using the template below:
```
Comment (lionelvillard) [inline docs/developer/eventing/sources/ping-source/README.md:124]: does `kn` support binary data?

Comment (snneji) [reply docs/developer/eventing/sources/ping-source/README.md:124]: I guessed that is does. I'm not sure if I'm right though. 

I looked at the [kn docs](https://github.com/knative/client/blob/main/docs/cmd/kn_source_ping_create.md#options) for the `kn source ping create` command and I found the option `--encoding base64` which I assumed was for users to apply binary data encoded in base64. 

 See line 167 in this PR for where I added this option.

Comment (snneji) [reply docs/developer/eventing/sources/ping-source/README.md:124]: @lionelvillard do you think I should remove the `kn` command for binary data?

Comment (lionelvillard) [reply docs/developer/eventing/sources/ping-source/README.md:124]: sounds good.

**Comment by omerbensaadon** [comment]
/lgtm

Super timely with all the Eventing 1.0 conversations happening

**Pull request merged**